ui/app/components/app-form/project-repository-settings.hbs

```diff
@@ -76,8 +76,8 @@
             @id="git-auth-not-set"
             name="git-auth-type"
             value={{0}}
-            checked={{eq this.authCase 0}}
-            {{on "input" (fn this.setAuthCase 0)}}
+            checked={{eq this.authCase 1}}
```

Comment:
I could be wrong, but `1` is not an authcase value

```
export enum AuthCase { 
      AUTH_NOT_SET = 0,
      BASIC = 4,
      SSH = 5,
    }
    ```

Reply:
I didn't realize that authCase was used elsewhere, I can change it but I ran into issues in `api.ts` on line 233 checking if we were trying to update the authentication method. I can update the logic to handle this differently by setting a default value for the `newAuthCase` to -1 or something else!